src/Modules/LearningTasks/Tutor.LearningTasks.API/Dtos/Activities/ExampleDto.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Tutor.LearningTasks.API.Dtos.Activities;
+
+public class ExampleDto
+{
+    public int Id { get; set; }
```

Comment:
Do we need an `Order` field for Examples?

Reply:
We don't need the order of examples.